neo4j-jdbc-http/src/main/java/org/neo4j/jdbc/http/driver/CypherExecutor.java

```diff
@@ -231,6 +238,28 @@ public Neo4jResponse executeQueries(List<Neo4jStatement> queries) throws SQLExce
 		return this.executeHttpRequest(request);
 	}
 
+	public List<String> callDbmsFunctions() {
+		try {
+			Neo4jResponse response = this.executeQuery(new Neo4jStatement(GET_DBMS_FUNCTIONS, Collections.emptyMap(), false));
+			if (response.hasErrors()) {
+				return Collections.emptyList();
+			}
+			return response.getResults()
```

Comment:
I'm thinking of introducing a `response.getSingleResult` here (same as `getFirstResult` but asserts that there is only 1 result... as it should be given there is only 1 query sent)
What do you think?

Reply:
It's a good idea but I would like to have all the efforts in the Bolt part, wdyt?